axum/src/response/sse.rs

```diff
@@ -202,11 +204,13 @@ impl Event {
     ///
     /// This corresponds to [`MessageEvent`'s data field].
     ///
-    /// [`MessageEvent`'s data field]: https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/data
+    /// Note that events with an empty data field will be ignored by the browser.
```

Comment:
Actually, is it possible to send an empty data field using `json_data`? Maybe this remark only makes sense on `data`.